Plotly.Blazor/wwwroot/plotly-interop-5.2.0.js

```diff
@@ -43,6 +43,14 @@ function onPlotlyReady(divId, callback) {
     }
 }
 
+function onPlotlyReadyFromChartData(callback) {
```

Comment:
Is that really required?  onPlotlyReady should be sufficient.

Reply:
Yes, sadly checking for the div id breaks the logic. In the new case there is no div id. I had no better idea than duplicating the function :(

Reply:
Ah, maybe passing a null divId could be handled in the original function and the check skipped.

Reply:
Fixed